nucypher/cli/painting.py

```diff
@@ -209,7 +211,7 @@ def paint_staged_stake(ursula,
 
     click.echo(f"""
 {ursula}
-~ Value      -> {stake_nu} NU ({stake_wei} NU-wei) 
+~ Value      -> {stake_value} ({Decimal(int(stake_value)):.2E} NU-Wei)
 ~ Duration   -> {duration} Days ({duration} Periods)
 ~ Enactment  -> {datetime_at_period(period=start_period)} (period #{start_period})
 ~ Expiration -> {datetime_at_period(period=end_period)} (period #{end_period})
```

Comment:
The `datetime` produced by `datetime_at_period()` is based on `maya.now()`, which means that it changes with time. Therefore, the datetimes in the `Stake` class and the ones printed here will differ. Perhaps `datatime_at_period` should receive an input `datetime` to avoid this.

Reply:
:thinking: - What would the input datetime be?

Reply:
Another approach here is to read on-chain period values from the `stake.owner.blockchain` and calculate from there.  

Ideally, we'd have an off-chain utility to calculate the current period, If possible (@szotov ), which Is what I've tried to build here.

Reply:
@cygnusv - Since we use floor division with seconds for period calculation, with periods based on epoch, wont they always be the same?